### PR TITLE
Multilingual Translation

### PR DESCRIPTION
**Pull request by siddbane10**
Cfreated multilingual translation feature

### CODE REVIEW

File: kairon/events/events.py [hunk 1]
```diff
@@ -146,3 +149,63 @@ def trigger_history_deletion(bot: Text, user: Text, month: int = 1, sender_id: T
         except Exception as e:
             logger.error(str(e))
             HistoryDeletionLogProcessor.add_log(bot, user, exception=str(e), status=EVENT_STATUS.FAIL.value)
+
+    @staticmethod
+    def trigger_multilingual_translation(bot: Text, user: Text, d_lang: Text = "en-US",
```
Comment (siddbane10) [inline kairon/events/events.py:154]: Remove default destination language and add appropriate test case

File: kairon/multilingual/processor.py [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+        :param translate_responses:
+        :return: new_bot_id: bot id of translated bot
+        """
+        bot_created = False
+        new_bot_id = None
+        domain = None
+        actions = None
+
+        try:
+            name = base_bot_name+'_'+d_lang
```
Comment (siddbane10) [inline kairon/multilingual/processor.py:189]: Do it with regex

File: kairon/multilingual/processor.py [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+                count += 1
+                name = base_bot_name+'_'+d_lang+'_'+str(count)
+
+            metadata = {
+                'source_language': s_lang,
+                'language': d_lang,
+                'source_bot_id': base_bot_id
+                }
+
+            new_bot = AccountProcessor.add_bot(name=name, account=self.account, user=self.user, metadata=metadata)
```
Comment (siddbane10) [inline kairon/multilingual/processor.py:201]: Create new bot after translations are completed

File: kairon/multilingual/processor.py [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+            new_bot_id = new_bot['_id'].__str__()
+            bot_created = True
+            logger.info(f"Created new bot with bot_id: {str(new_bot_id)}")
+
+            nlu = self.__translate_training_data(base_bot_id=base_bot_id, s_lang=s_lang, d_lang=d_lang)
+            logger.info("Translated training data successfully.")
+
```
Comment (siddbane10) [inline kairon/multilingual/processor.py:208]: load all the source bot data before calling everything

**Review by sfahad1414 — APPROVED** [review]
Approved

**Pull request merged**